docs/source/examples/spot-jobs.rst

```diff
@@ -175,3 +202,17 @@ Here are some commands for managed spot jobs. Check :code:`sky spot --help` for
     # Cancel a spot job by name
     $ sky spot cancel -n bert-qa
 
+
+Advanced note (spot controller)
+-------------------------------
+
+There will be a single spot controller cluster (a small on-demand CPU VM) running in the background to manage all the spot jobs.
```

Comment:
> There will be a single spot controller cluster

Should we say cluster? because it's only a single server. just worried ppl may confuse cluster with multiple servers.

Reply:
Good point! Changed to VM instead.